tools/sdk-generation-pipeline/scripts/install-vscode-server.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/sh
+set -e
+
+commit_sha="da15b6fd3ef856477bf6f4fb29ba1b7af717770d"
```

Comment:
will it change with VS Code Server update? consider config it with parameter? just a suggestion

Reply:
This commit_sha is only used to install a vscode server and use the vscode server to install extension. If the server doesn't work for your local vscode, you vscode will update it when connecting to the docker. So, it's no need to config it. Also, it's used in building docker image, we also cannot config it.